### ISSUE TITLE
Is ChaiScript alive?

### ISSUE THREAD

**Issue by YarikTH** [ChaiScript/ChaiScript#532]
As I see last release was 2 years ago, 151 commits since then. But CI builds are broken, and almost no activity on the codebase, issues etc.
I didn't see any mentions about repository state in the chai script site or in any obvious place.
Are there any plans to continue the script support or this repo is factically abandoned?

**Comment by georgelam6** [comment]
I think Jason has moved on from ChaiScript now, which is understandable, considering the amount of time he's already put into this project.

**Comment by RobLoach** [comment]
He's given commit access to a few people! If anyone has any PRs to push forwards, I'd be more than happy to help merge. I think the biggest issues are what you listed above.... Fix tests, update the CI, ensure C++20 compat, etc.

**Comment by lefticus** [comment]
Correct, I'm not actively maintaining this, but @RobLoach has commit access. The C++17 work on develop could probably be merged into master at some point. I last interest after 10+ years of working on it followed by 2+ years of waiting for compilers to support C++17 features. I think at some point in 2019 it was possible to support the 3 major compilers with the C++17 features I was using.

**Comment by patlecat** [comment]
@lefticus 
Would be so cool. What updates have you been waiting for in C++17 for Chaiscript? And how would that benefit Chaiscript?

**Comment by totalgee** [comment]
Ping... Hi @RobLoach, just wondering if there is any possibility of moving forward with resolving some pull requests, like the [VS2019 branch](https://github.com/ChaiScript/ChaiScript/pull/539)?

(BTW, the most recent two PRs look like spam and can be rejected/closed...)
